src/shared/install.h

```diff
@@ -48,6 +48,11 @@ enum UnitFileFlags {
         _UNIT_FILE_FLAGS_MASK_PUBLIC = UNIT_FILE_RUNTIME|UNIT_FILE_PORTABLE|UNIT_FILE_FORCE,
 };
 
+enum CopyFileFlags {
+        COPY_FILE_REPLACE = 1 << 0, /* Public API via DBUS, do not change */
+        _COPY_FILE_FLAGS_MASK_PUBLIC = COPY_FILE_REPLACE,
+};
```

Comment:
I'd put this in machine-dbus.h for now since it's specific to the machined D-Bus methods